puppet/zulip/templates/postgresql/11/postgresql.conf.template.erb

```diff
@@ -0,0 +1,715 @@
+# -----------------------------
```

Comment:
This should be done as 2 commits; first one checking in the upstream default copy and then a second adding our changes.

Reply:
@timabbott, I'm sorry, but I didn't understand what you were suggesting here. For example, what "upstream" default copy are you referring to and where would the check exist?

Reply:
Run `git log` on the similar file for postgres 10 and you'll see what I have in mind.